Dockerfile

```diff
@@ -14,6 +14,7 @@ COPY package-lock.json package-lock.json
 COPY --chown=app:app . /app
 
 RUN npm ci --audit=false && rm -rf ~app/.npm /tmp/*
+RUN npm run build
```

Comment:
Is this the correct time to run? (Before setting ENVs)

Reply:
Actually you are right, fixed